javascript/challenges/stacksAndQueues/fifoAnimalShelter/README.md

```diff
@@ -0,0 +1,17 @@
+# fifo animal shelter
+
+Create a class called AnimalShelter which holds only dogs and cats. The shelter operates using a first-in, first-out approach.
+Implement the following methods:
+enqueue(animal): adds animal to the shelter. animal can be either a dog or a cat object.
+dequeue(pref): returns either a dog or a cat. If pref is not "dog" or "cat" then return null.
+
+## Challenge 12
+
+### Approach & Efficiency
+
+enqueue is o1 because you're adding 1 obj to the list of objects
+dequeue is also o1 because you're removing and returning one object
+
+### API
+
+[whiteboard](../../../assets/class-12.jpg)
```

Comment:
You're off to a great start! If you have any questions still lingering about the problem domain let me know! Once you have tests added I will regrade for you!